ios/gutenberg/GutenbergViewController.swift

```diff
@@ -0,0 +1,30 @@
+
+import UIKit
+
+class GutenbergViewController: UIViewController {
```

Comment:
Going the route of providing a VC in Gutenberg is a double edged sword, and one that I'd really try to avoid, if possible.

I realize I may not see the full picture so my initial question is: was this choice because of advantages?  requirements?

To explain a bit more why I think this is a bigger deal that can be apparent at first... this is all coming from my experience working in Aztec:

- This couples the UI surrounding the editor, to `gutenberg-mobile`.  If the UI needs to be customized we'll need to add extra code to support any such customization (that'd be otherwise for free).
- This potentially couples other things too, such as service calls, business logic (that may already exist WPiOS).  We may also need to add native dependencies to `gutenberg-mobile` (things like `WordPressShared`... the media provider... services... UI components that exist in WPiOS... etc).
- It would reduce our ability to reuse Gutenberg elsewhere.  I don't have a clear picture of where Gutenberg may end up showing up, but the advantage in working the editor as a view component is that you can place it anywhere.

Overall, while it may seem simpler, it may end up adding a ton of extra work to access things that we'd get for free if we just provided a view component.

Reply:
Thanks for your comments @diegoreymendez !

I believe there's a misunderstanding. Probably I haven't explained this implementation properly.
This `GutenbergViewController` only exists in the Example app and is meant to be a testing platform for the native modules (the communication between RN and Native).

The plan (as I see it so far) would be to provide just the `Gutenberg.h` interface to the `gutenberg-mobile` users (i.e: WPiOS). `Gutenberg.h` currently lives inside `react-native-gutenberg-bridge` but ideally we would provide it as the public header in a Pod framework. It provide access to the Gutenberg RootView (plain `UIView`), and it doesn't have any React / ReactNative dependencies. It also has methods to send messages to the RN side and a delegate to receive messages from RN.

Reply:
Maybe we could rename it `GutenbergExampleController`, or plane `ExampleController` to be more explicit about it.

Reply:
I think my confusion stems from the fact there's no mention of `Example` in the file path or name... it's not a big deal to be honest.  If this is an Example App VC we should be good.